app-mobilephone/gammu/gammu-1.39.0-r1.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
the switch to EAPI-7 is not complete here. save this for the `1.40.0` bump.

Reply:
why not complete? no warning from repoman, cmake-utils EAPI=7 ok.

Reply:
But repoman has never given porting instructions, right? That's just not what it does. for a proper EAPI-7 bump you have to look at `DEPEND` and check what should be moved into `BDEPEND` instead.

Reply:
OK, I got it.